boilr.go

```diff
@@ -1,7 +1,18 @@
 package main
 
-import "github.com/tmrts/boilr/pkg/cmd"
+import (
+  "github.com/tmrts/boilr/pkg/boilr"
+  "github.com/tmrts/boilr/pkg/cmd"
+  "github.com/tmrts/boilr/pkg/util/exit"
+  "github.com/tmrts/boilr/pkg/util/osutil"
+)
 
 func main() {
-	cmd.Run()
+  if exists, _ := osutil.DirExists(boilr.Configuration.TemplateDirPath); ! exists {
+    if err := osutil.CreateDirs(boilr.Configuration.TemplateDirPath); err != nil {
+      exit.Error(err)
```

Comment:
If it returns an error, it will be something related to directory creation. Users will be confused to get a directory error out of the blue, we should send an info message here explaining that it wasn't init'ed and now its being initialized